src/nanovg.c

```diff
@@ -2545,11 +2545,13 @@ int nvgTextBreakLines(NVGcontext* ctx, const char* string, const char* end, floa
 					breakMaxX = 0.0;
 				}
 			} else {
-				float nextWidth = iter.nextx - rowStartX;
```

Comment:
Hmm I don't think my communication is good here. I'll show you the issue when I get some time.